src/main/java/microsoft/exchange/webservices/data/ExchangeServiceBase.java

```diff
@@ -98,19 +96,16 @@
   /**
    * The requested server version.
    */
-  private ExchangeVersion requestedServerVersion =
-      ExchangeVersion.Exchange2010_SP2;
+  private ExchangeVersion requestedServerVersion = ExchangeVersion.Exchange2010_SP2;
 
   /**
    * The server info.
    */
   private ExchangeServerInfo serverInfo;
 
-  private Map<String, String> httpHeaders = new HashMap<String, String>();
-
-  private Map<String, String> httpResponseHeaders = new HashMap<String, String>();
+  private Map<String, String> httpHeaders = new HashMap<>();
 
-  private TimeZone timeZone;
+  private Map<String, String> httpResponseHeaders = new HashMap<>();
```

Comment:
I reverted this specific change, but the project settings indicate I can use Java 1.7 features, but Maven is not set up correctly. I have created a separate issue for this (#167).